tests/Aspire.Hosting.Keycloak.Tests/KeycloakPublicApiTests.cs

```diff
@@ -107,26 +107,26 @@ public void WithRealmImportShouldThrowWhenBuilderIsNull()
     }
 
     [Fact]
-    public void WithRealmImportShouldThrowWhenImportDirectoryIsNull()
+    public void WithRealmImportShouldThrowWhenImportIsNull()
     {
         var builder = TestDistributedApplicationBuilder.Create();
         var keycloak = builder.AddKeycloak("Keycloak");
-        string importDirectory = null!;
+        string import = null!;
 
-        var action = () => keycloak.WithRealmImport(importDirectory);
+        var action = () => keycloak.WithRealmImport(import);
 
         var exception = Assert.Throws<ArgumentNullException>(action);
-        Assert.Equal(nameof(importDirectory), exception.ParamName);
+        Assert.Equal(nameof(import), exception.ParamName);
     }
 
     [Fact]
-    public void WithRealmImportShouldThrowWhenImportDirectoryDoesNotExist()
+    public void WithRealmImportShouldThrowWhenImportDoesNotExist()
```

Comment:
Seems we have no tests that actually assert the happy path behavior! Could add a test/tests to verify that calling it with a directory or file that **does** exist adds the expected annotation?

Reply:
Added!